src/components/About.jsx

```diff
@@ -1,117 +1,61 @@
-    display:flex;
-    
-    justify-content: stretch;
-    align-self: center;
-    border-top: 1px solid rgba(0,0,0,0.3);
-    
-   
-    padding: 2em;
-    /* height: 50em; */
-`;
-
-const Icon = styled.div`
-    width: 50px;
-    height: auto;
-    margin: 0 2em;
-`;
-function About() {
+import React from "react";
+import "./assets/css/components/about.us.css";
+import Team from "./assets/images/undraw_team_spirit_hrr4.svg";
+//Icons
+import Facebook from "./assets/images/facebook.svg";
+import Github from "./assets/images/github.svg";
+import Instagram from "./assets/images/instagram.svg";
+import Linkdin from "./assets/images/linkedin.svg";
+import Twitter from "./assets/images/twitter.svg";
+
+const AboutUs = () => {
     return (
-        <Wrapper>
-           <h1 className="text-center">About</h1>
-            <Content>
-            <BackgroundImage  src={aboutSvg}/>
-                <Text>
-                    There are many variations of passages of Lorem Ipsum available, 
-                    but the majority have suffered alteration in some form, by injected 
-                    humour, or randomised words which don't look even slightly believable. 
-                    If you are going to use a passage of Lorem Ipsum, you need to be sure there 
-                    isn't anything embarrassing hidden in the middle of text. All the Lorem Ipsum
-                    generators on the Internet tend to repeat predefined chunks as necessary, making 
-                    this the first true generator on the Internet. It uses a dictionary of over 200
-                    Latin words, combined with a handful of model sentence structures, to generate 
-                    Lorem Ipsum which looks reasonable. The generated 
-                    Lorem Ipsum is therefore always free from repetition, injected humour, or non-characteristic words etc.
-                </Text>
-                
-              
-            </Content>
-            <Logos>
-                    <Icon><img src={instagramLogo} alt='Instagram Logo'/></Icon>
-                    <Icon><img src={whatsappLogo} alt='WhatsApp Logo'/></Icon>
-                    <Icon><img src={githubLogo} alt='GitHub Logo'/></Icon>
-            </Logos>
-        </Wrapper>
+        <div>
+            <div className="container-fluid heading-background">
+                <div className="row">
+                    <div className="text-center col-12 col-md-12 col-sm-12 my-5">
+                        <h1 className="heading">The Social Scheduler</h1>
+                        <p className="tag">Lorem Ipsum has been the industry's standard dummy</p>
+                    </div>
+                </div>
+            </div>
+            <div className="container about-backgroud">
+                <div className="row">
+                    <div className="col-6 col-md-6 col-sm-6 my-5">
+                        <img src={Team} alt="team" className="team-image"/>
+                    </div>
```

Comment:
Put comments. Document each section of your code. Readability of code becomes a key factor to the success of a project in later phases

Reply:
okay